sources/packages/backend/workflow/src/workflow-definitions/load-assessment-consolidated-data.bpmn

```diff
@@ -108,6 +108,8 @@
           <zeebe:header key="studentDataAdditionalTransportCost" value="$.data.additionalTransportCost" />
           <zeebe:header key="studentDataAdditionalTransportWeeks" value="$.data.additionalTransportWeeks" />
           <zeebe:header key="studentDataAdditionalTransportPlacement" value="$.data.additionalTransportPlacement" />
+          <zeebe:header key="applicationStatus" value="$.applicationStatus" />
+          <zeebe:header key="applicationNOAApproval" value="$.noaApproval" />
```

Comment:
Following this https://github.com/bcgov/SIMS/pull/3332/files#r1614098561, would be `applicationHasNOAApproval`.

Reply:
Updated